### DISCUSSION TITLE
Displaying form error from API

### DISCUSSION THREAD

**Discussion by xtecneo** [ant-design/ant-design#39727] [Q&A]
I'm developing a registration form, where after the user submits the form system will check whether the email is already registered or not so when the email isRegistered I want to set the email field to be error state, but I can't find a method that looks like setFieldError in the form

**Comment by bombillazo** [comment]
Hey, you need to use the `setFields` function:

```tsx
 form.setFields([
      {
        errors: ['Please select an option for each row', 'My custom error'],
        name: 'my-field',
      },
    ]);
```
